docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1307,6 +1307,26 @@ How to **investigate** and **fix** it:
     - Check the number of in-memory series shown on the `Mimir / Tenants` dashboard for an approximation of the number of series that will be compacted once these blocks are shipped from ingesters.
     - Check the configured `compactor_split_and_merge_shards` for the tenant. A reasonable rule of thumb is 8-10 million series per compactor shard - if the number of series per shard is above this range, increase `compactor_split_and_merge_shards` for the affected tenant(s) accordingly.
 
+## Mimir ingest storage (experimental)
+
+This section contains runbooks for alerts related to experimental Mimir ingest storage.
+In this context, any reference to Kafka means a Kafka protocol-compatible backend.
+
+### IngesterLastConsumedOffsetCommitFailed
+
+This alert is fired when an ingester is failing to commit the last consumed offset to the Kafka backend.
+
+How it **works**:
+
+- The ingester ingests data (metrics, exemplars, ...) from Kafka and periodically commits the last consumed offset back to Kafka.
+- At startup, an ingester reads the last consumed offset committed to Kafka and resume the consumption from there.
```

Comment:
```suggestion
- At startup, an ingester reads the last consumed offset committed to Kafka and resumes the consumption from there.
```

Reply:
Fixed in https://github.com/grafana/mimir/pull/7699/commits/324739bef4c55fec0d016fc3d0248c432464eff1